### ISSUE TITLE
Findable Help SC3.2.6 - all changes together

### ISSUE THREAD

**Issue by guyhickling** [w3c/wcag#1242]
If the changes suggested in #1228 - #1232 are put together, they would change SC3.2.6 to the following:
> For single page Web applications or any set of Web pages, if one of the following four help sources is provided in the web pages, then access to at least one source is included, in the same position relative to other items, on each page:
⦁	Human contact details;
⦁	Human contact mechanism;
⦁	Help provided by the website or application;
⦁	A fully automated contact mechanism.

This is much clearer and easier to understand than the current version (from the July 21 Editor's draft). That said:

_For single page Web applications or any set of Web pages, if one of the following is available, then access to at least one option is included in the same relative order on each page:
⦁	Human contact details;
⦁	Human contact mechanism;
⦁	Self-help option;
⦁	A fully automated contact mechanism._

**Comment by SteveALee** [comment]
@guyhickling 

Thanks for the suggestion. I think you raise 2 separate points here. These are my thoughts:

## It's hard to read. 

I'm not conviced the suggestion is so easier to read.

* "is available" -> " four help sources is provided in the web pages" - "Available" covers any help service the site owner may have, not just those they may already list on the pages.  In other words, help is important and should be listed on the pages, This is especially the case for users with cognitive accessibility requirements. We might want to clarify this point.
* "option" -> "source" - perhaps marginally clearer when paired with a similar change earlier in the sentence

The compound sentence makes it a little hard to parse so I might be tempted to separate it out thus, at the expense of making it longer:

"in the same relative order on each page" -> "When multiple options are provided, at least one is always provided on each page.  Multiple options are always listed in the same relative order"

## "Self-help option" -> "Help provided by the website or application"

 I do think this is better given self-help could be interpreted to mean "Google it" which is definitely *not* the intention.

**Comment by guyhickling** [comment]
Steve,
Thank you for your agreement on the self-help item being ambiguous. Hopefully they will change it.

You also say "The compound sentence makes it a little hard to parse....", and I agree with you. However in suggesting this series of changes to this SC I have tried to disturb the layout and wording as little as possible, concentrating only on recommending changes to things that either make no sense at all to the ordinary (i.e. non-WCAG) person, or are ambiguous, or are logically at fault. 

Believe me when I say it is difficult enough trying to get anyone involved to even consider any suggestions for improvement. The default reaction throughout the WCAG 2.2 public commenting has been to object to every suggestion for improvement raised. So I haven't attempted to suggest more than the minimum changes needed to achieve some level of comprehensibility. Trying to reword things to make them even more understandable to the ordinary reader would be just a step too far. Of course, It's why the WCAG has the reputation it has for being incomprehensible!

**Comment by alastc** [comment]
@guyhickling the usual reason for not taking on suggestions is that it changes the scope. If there is a simpler way to say something, that _has the same scope_, that would be great. It doesn't often happen though.

**Comment by guyhickling** [comment]
@alastc  If a suggestion intended to make an SC more understandable and meaningful has the effect of changing the scope then, in the interests of making the WCAG clearer and less abstruse, surely the proper response should be to try to help, not simply refuse to entertain the idea! Point out that it changes the scope, and offer an amendment to the suggestion that continues trying to make the SC more understandable without changing the scope.

In issue #1231 this is exactly what happened. The suggestion I made there did have the unfortunate effect of changing the scope - inadvertently, not by intention. @JAWS-test immediately pointed out the error, so I amended the suggested wording so it did not change the meaning or scope. What happened there was constructive criticism, which is the way things should be. 

Unfortunately this is not what has usually happened in the public comments on 2.2. Most suggestions on these new SCs have been immediately rejected outright, regardless of the improvements that could be made. That does not achieve anything useful for anyone - least of all for the disabled people we are trying to help, and certainly not for the developers that will have to try to understand these new SCs and try to work out what on earth they mean.

Now I ask you, looking at the suggested new wording for SC3.2.6 at the top of this issue (and also taking into account issues #1228 to #1232 where the reason for each suggested change is explained in detail): in what way has the scope been changed? All they do is to take a totally incomprehensible new SC, and try to make it more comprehensible. But so far no one has even bothered to consider any of those issues.

**Comment by alastc** [comment]
Hi @guyhickling,

I think there's a misunderstanding of process going on, particularly on SC 3.2.6 where the issues have not even been discussed by the working group (WG) yet.

We've a fairly large backlog (see the [project page filtered to 3.2.6](https://github.com/w3c/wcag/projects/2?card_filter_query=label%3A%223.2.6+Findable+help%22)). Once individuals in the WG have read through the issues and proposed a change or response, we group those together and discuss them at a meeting.

If you compare that view to the [project page filtered to focus appearance](https://github.com/w3c/wcag/projects/2?card_filter_query=label%3A%222.4.11+Focus+appearance+%28min%29%22), you'll see a lot more in the 'done' column for that SC because we've processed more comments.

You are one of 6 people who've raised issues on 3.2.6, and we will deal with all the issues. 

However, before it goes to the group, various people (public and WG) will enter their views. That is not an ‘immediate rejection’. No one gets to do that, including me (as co-chair). It might well be a disagreement, but within github it is all individual views unless marked otherwise.

It is helpful to take on board other people’s views, they may well have thought about this SC for a long time and have more context. Once a thread has reached some sort of conclusion it gets added to an agenda for the WG to come to a decision.

In this case @scha14 has assigned these issues to herself, so I expect she will either propose changes based on your comments, or propose a response outlining why she doesn't think there should be changes. Whatever she proposes will go to the group, who will decide on the outcome.

**Comment by guyhickling** [comment]
@alastc  Thank you for that. The backlog explains the lack of comments to the 3.2.6 issues - after a month with no attention it looked as though they were going to slip through the net! In that connection, what is the expected 2.2 timetable now? From what I remember (which is a bit hazy now) of the original schedule, it seems to have slipped somewhat. Not surprising really, considering the pandemic and everything else. So what happens after the wide review?

**Comment by alastc** [comment]
After wide review we ensure we've processed all the comments, then take stock of the changes and decide if another wide review is needed (rinse & repeat). If not, we go through to Candidate Recommendation stage, where it's reviewed by W3C members and we have to make sure there are implementations for everything. See the [process doc (short version)](https://github.com/w3c/wg-effectiveness/blob/master/process.md#user-content-w3c-recommenation-technical-report-flow-the-recommendation-track) for more.

**Comment by scha14** [comment]
All changes above are addressed in their individual issues for now. Once I hear from @guyhickling about closing one or more of those, we can bring the remaining discussion here.

**Comment by scha14** [comment]
Copying response from https://github.com/w3c/wcag/issues/1231 that reporters agree addresses all issues except one suggestion which changes it to 

> For single page Web applications or any set of Web pages, if one or more of the following ways of finding help is provided, then access to at least one way of finding help is included, **appearing** in the same relative order on each page:

Below is the original response :

Hi everyone, we discussed this in a meeting and whilst agreeing with the substance of the comment, it's a hard terminology problem.

We tried: adding 'option' to both ends, and tried the terms mechanism, method, feature, means and ways. Each of these, particularly mechanism/method/feature imply things we do not intend.

We'd like to propose going with this as the first part of the SC text:

> For single page Web applications or any set of Web pages, if one or more of the following ways of finding help is provided, then access to at least one way of finding help is included in the same relative order on each page:

**Comment by scha14** [comment]
Draft pull request 
https://github.com/w3c/wcag/pull/1482

**Comment by alastc** [comment]
Hi @scha14,

There have been some overlapping changes going on, I merged in a previous PR (approved but I must have missed it) and the current text is:

> "For <a>single page Web applications</a> or any <a>set of Web pages</a>, if one or more of the following ways of finding help is supported, then access to at least one way of finding help is included in the same relative order on each page:"

That is exactly the same as PR 1482, except the PR removes "or more" from "if one or more of the...".

I think we need the "or more", otherwise sites which have more than one don't have to meet the SC.

For anyone else on the thread, does that cover it? (Given the other discussions.)

**Comment by scha14** [comment]
@guyhickling is this good to close with the latest changes?

**Comment by alastc** [comment]
I believe the [latest version](https://w3c.github.io/wcag/understanding/findable-help.html) covers this (and includes other updates), so closing.
